pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -412,6 +412,12 @@ func (r *Reconciler) runReconcileSeedFlow(
 			Dependencies: flow.NewTaskIDs(syncPointReadyForSystemComponents),
 			SkipIf:       seedIsGarden,
 		})
+		_ = g.Add(flow.Task{
+			Name:         "Deploying gardener-custom-metrics",
+			Fn:           c.gardenerCustomMetrics.Deploy,
+			Dependencies: flow.NewTaskIDs(syncPointReadyForSystemComponents),
+			SkipIf:       seedIsGarden,
```

Comment:
Hmm, this component is not deployed by the garden operator so if a seed cluster is both a garden cluster (a `garden` resource exists) no one will currently deploy the `gardener-custom-metrics`. Same for the delete flow.

Reply:
@rfranzke wrt https://github.com/gardener/gardener/pull/9570#discussion_r1562366998, this `SkipIf` should now be removed and re-added once GOP is extended to also deploy GCMx, right?

Reply:
Yes, right

Reply:
@plkokanov Thanks for catching this!

Fixed and will be available with next push.